scan-triggers/aws-python-bucket-full-and-scheduled-scan/template.yaml

```
@@ -0,0 +1,902 @@
+Parameters:
+  BucketName:
+    Type: String
+    Description: Name of a bucket that you want to full scan. Make sure you have FSS Storage Stack deployed around it already.
+  ScannerQueueArn:
+    Type: String
+    Description: ARN of the ScannerQueue queue. Something like arn:aws:sqs:us-east-1:123456789012:All-in-one-TM-FileStorageSecurity-ScannerStack-IT1V5O-ScannerQueue-1IOQHTGGGZYFL
+  ScannerQueueUrl:
+    Type: String
+    Description: URL of the ScannerQueue queue. Something like https://sqs.us-east-1.amazonaws.com/123456789012/All-in-one-TM-FileStorageSecurity-ScannerStack-IT1V5O-ScannerQueue-1IOQHTGGGZYFL
+  ScanResultTopicArn:
+    Type: String
+    Description: ARN of ScanResultTopic topic. Something like arn:aws:sns:us-east-1:123456789012:All-in-one-TM-FileStorageSecurity-StorageStack-1E00QCLBZW7M4-ScanResultTopic-1W7RZ7PBZZUJO
+  Schedule:
+    Type: String
+    Default: ""
+    Description: "Set a schedule for full scan. If empty, there will not be a scheduled scan. Defaults to empty. More info at: https://docs.aws.amazon.com/lambda/latest/dg/services-cloudwatchevents-expressions.html"
+Conditions:
+  SetSchedule:
+    Fn::Not:
+      - Fn::Equals:
+          - ""
+          - Ref: Schedule
+Resources:
+  StateBucket446E0578:
```

Would you please remove the random hex suffix in logical IDs of all resources?
I think it would make the template easier to read and understand.
```suggestion
  StateBucket:
```